integration/tests/access/execution_state_sync_test.go

```diff
@@ -0,0 +1,209 @@
+
+	bridgeID flow.Identifier
+	ghostID  flow.Identifier
+
+	// root context for the current test
+	ctx    context.Context
+	cancel context.CancelFunc
+
+	net *testnet.FlowNetwork
+}
+
+func (s *ExecutionStateSyncSuite) SetupTest() {
+	logger := unittest.LoggerWithLevel(zerolog.InfoLevel).With().
+		Str("testfile", "unstaked.go").
+		Str("testcase", s.T().Name()).
+		Logger()
+
+	s.log = logger
+	s.log.Info().Msg("================> SetupTest")
+	s.ctx, s.cancel = context.WithCancel(context.Background())
+
+	s.buildNetworkConfig()
+
+	// start the network
+	s.net.Start(s.ctx)
+
+	s.Track(s.T(), s.ctx, s.Ghost())
+}
+
+func (s *ExecutionStateSyncSuite) TearDownTest() {
+	s.log.Info().Msg("================> Start TearDownTest")
+	s.net.Remove()
+	s.cancel()
+	s.log.Info().Msgf("================> Finish TearDownTest")
+}
+
+func (s *ExecutionStateSyncSuite) Ghost() *client.GhostClient {
+	ghost := s.net.ContainerByID(s.ghostID)
+	client, err := lib.GetGhostClient(ghost)
+	require.NoError(s.T(), err, "could not get ghost client")
+	return client
+}
+
+func (s *ExecutionStateSyncSuite) buildNetworkConfig() {
+	// staked access node
+	s.bridgeID = unittest.IdentifierFixture()
+	bridgeANConfig := testnet.NewNodeConfig(
+		flow.RoleAccess,
+		testnet.WithID(s.bridgeID),
+		testnet.SupportsUnstakedNodes(),
+		testnet.WithLogLevel(zerolog.DebugLevel),
+		testnet.WithAdditionalFlag("--execution-data-sync-enabled=true"),
+		testnet.WithAdditionalFlag(fmt.Sprintf("--execution-data-dir=%s", testnet.DefaultExecutionDataServiceDir)),
+		testnet.WithAdditionalFlag("--execution-data-retry-delay=1s"),
+	)
+
+	// add the ghost (access) node config
+	s.ghostID = unittest.IdentifierFixture()
+	ghostNode := testnet.NewNodeConfig(
+		flow.RoleAccess,
+		testnet.WithID(s.ghostID),
+		testnet.WithLogLevel(zerolog.FatalLevel),
+		testnet.AsGhost())
+
+	consensusConfigs := []func(config *testnet.NodeConfig){
+		testnet.WithAdditionalFlag("--hotstuff-timeout=12s"),
+		testnet.WithAdditionalFlag("--block-rate-delay=100ms"),
+		testnet.WithAdditionalFlag(fmt.Sprintf("--required-verification-seal-approvals=%d", 1)),
+		testnet.WithAdditionalFlag(fmt.Sprintf("--required-construction-seal-approvals=%d", 1)),
+		testnet.WithLogLevel(zerolog.FatalLevel),
+	}
+
+	net := []testnet.NodeConfig{
+		testnet.NewNodeConfig(flow.RoleCollection, testnet.WithLogLevel(zerolog.FatalLevel)),
+		testnet.NewNodeConfig(flow.RoleCollection, testnet.WithLogLevel(zerolog.FatalLevel)),
+		testnet.NewNodeConfig(flow.RoleExecution, testnet.WithLogLevel(zerolog.FatalLevel)),
+		testnet.NewNodeConfig(flow.RoleExecution, testnet.WithLogLevel(zerolog.FatalLevel)),
+		testnet.NewNodeConfig(flow.RoleConsensus, consensusConfigs...),
+		testnet.NewNodeConfig(flow.RoleConsensus, consensusConfigs...),
+		testnet.NewNodeConfig(flow.RoleConsensus, consensusConfigs...),
+		testnet.NewNodeConfig(flow.RoleVerification, testnet.WithLogLevel(zerolog.FatalLevel)),
+		bridgeANConfig,
+		ghostNode,
+		// TODO: add observer
```

Comment:
Update this?

Reply:
I'm waiting for the changes from Miklos that adds support for observers in testnet

Reply:
Okay, can we add some more detail to this comment or create an issue?

Reply:
https://github.com/onflow/flow-go/issues/2392